examples/cp-all-in-one-cloud/ccloud-generate-env-vars.sh

```diff
@@ -0,0 +1,57 @@
+#!/bin/bash
+
```

Comment:
Copyright?

Reply:
@edenhill , can you please provide example file that has the copyright to use?

Reply:
https://github.com/confluentinc/cp-docker-images/blob/489001af7ba802fce3fe4fd2f149b9061ae585a4/debian/enterprise-control-center/include/etc/confluent/docker/launch